pkg/providers/vsphere/upgrade/virtualmachine/vm_schema_upgrade.go

```diff
@@ -23,6 +25,10 @@ import (
 	"github.com/vmware-tanzu/vm-operator/pkg/util/ptr"
 )
 
+const (
+	skipDisksReconciliationMsg = "Skipping disks reconciliation due to %s"
```

Comment:
Can we just put the full message in the log? It is nice to be able to just search for the whole error message